scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -2575,7 +2577,17 @@ void BPF_STRUCT_OPS(layered_stopping, struct task_struct *p, bool runnable)
 
 	if (cpuc->yielding && runtime < task_layer->slice_ns)
 		runtime = task_layer->slice_ns;
-	p->scx.dsq_vtime += runtime * 100 / p->scx.weight;
+
+	hint = 1024;
+	task_hint = bpf_task_storage_get(&scx_layered_task_hint_map, p, NULL, 0);
+	if (task_hint)
+		hint = task_hint->hint > 1024 ? 1024 : task_hint->hint;
+	hint = hint ?: 1;
```

Comment:
Can this be in the body of the preceding `if` block. Range limiting spread like this reads weird.